components/mpas-ocean/src/shared/mpas_ocn_gm.F

```diff
@@ -910,6 +917,17 @@ subroutine ocn_GM_init(domain, err)!{{{
             sfcTaperFactor = 0.0_RKIND
          end if
 
+         if (config_gm_gravWaveSpeed=='constant') then
+            gm_constant_gravWaveSpeed = config_gm_constant_gravWaveSpeed
+            gm_computed_gravWaveSpeed_on = 0.0_RKIND
+         elseif (config_gm_gravWaveSpeed=='computed') then
+            gm_constant_gravWaveSpeed = 0.0_RKIND
+            gm_computed_gravWaveSpeed_on = 1.0_RKIND
```

Comment:
instead of these real values, why not just have the block switch `local_config_GM_lat_variable_c2` to .true. or ..false. and then remove from the config_GM_closure blocks below?